docs/src/main/sphinx/admin/opentelemetry.md

```diff
@@ -0,0 +1,17 @@
+# Observability with OpenTelemetry
+
+Trino exposes tracing information for observability monitoring of a running
+Trino deployment for the widely used [OpenTelemetry](https://opentelemetry.io/)
+collection of APIs, SDKs, and tools. Use it to instrument, generate, collect,
+and export telemetry data such as metrics, logs, and traces to help you analyze
```

Comment:
We support only OTEL tracing

Reply:
Haha .. this not even a draft yet .. but I will update. Thanks @wendigo